application/controllers/classification_type_controller.py

```diff
@@ -52,3 +52,26 @@ def get_classification_types():
         return [classification.to_dict() for classification in classification_types]
 
     logger.info("Attempted to retrieve classification types when none in database")
+
+
+def _delete_classification_type(classification_type):
+    try:
+        deleted_classification_types = db.session.query(ClassificationType).filter(ClassificationType.name ==
```

Comment:
I believe .delete() returns the row count, so I would prefer the variable to be called similar